tests/cli/commands/test_plan.py

```diff
@@ -0,0 +1,31 @@
+from tests.cli.commands import invoke, config_path, run_properties_path
+
+
+class TestPlanCli:
+    def test_plan_create(self):
```

Comment:
```suggestion
    def test_plan_create_help(self):
```

Reply:
It's not super trivial, but actually what I'm testing here is whether the `mpyl plan create` commands works. The fact that I'm calling `--help` is a detail (mostly because I don't want/need to run the entire thing).